sklearn/metrics/tests/test_regression.py

```diff
@@ -56,6 +56,17 @@ def test_regression_metrics(n_samples=50):
                         np.sum(1 / y_true) / (4 * n))
 
 
+def test_mean_squared_error_multioutput_raw_value_squared():
+    mse1 = mean_squared_error(
+        [[1]], [[10]], multioutput="raw_values", squared=True
+    )
+    mse2 = mean_squared_error(
+        [[1]], [[10]], multioutput="raw_values", squared=False
+    )
+    with pytest.raises(AssertionError):
+        assert_allclose(mse1, mse2)
```

Comment:
mse1 and mse2 should be different here. `np.sqrt(mse1)` should equal `mse2`.

Reply:
I have checked that assert `mse1==mse2` raises `AssertionError` or not here. if it doesn't raise that error then the test will fail.

Reply:
Ah right, my bad, looks good then!